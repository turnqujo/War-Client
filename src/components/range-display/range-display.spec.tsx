import { shallow } from 'enzyme';
import React from 'react';
import RangeDisplay from './range-display';

describe('The Range display Component', () => {
  test('renders without crashing.', () => {
    shallow(<RangeDisplay start={1} end={10} />);
  });

  test('should match its snapshot.', () => {
    const wrapper = shallow(<RangeDisplay start={1} end={10} />);
    expect(wrapper).toMatchSnapshot();
  });
});
