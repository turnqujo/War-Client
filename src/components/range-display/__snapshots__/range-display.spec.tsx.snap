// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`The Range display Component should match its snapshot. 1`] = `ShallowWrapper {}`;
